x/programs/examples/storage/storage.go

```diff
@@ -17,25 +17,25 @@ const (
 	programPrefix = 0x0
 )
 
+// ProgramPrefixKey returns a properly formatted key
+// for storing a value at [id][key].
 func ProgramPrefixKey(id []byte, key []byte) (k []byte) {
-	k = make([]byte, consts.IDLen+1+len(key))
+	k = make([]byte, 1+consts.IDLen+len(key))
 	k[0] = programPrefix
-	copy(k, id[:])
-	copy(k[consts.IDLen:], (key[:]))
+	copy(k[1:1+consts.IDLen], id[:])
```

Comment:
Don't need the 1+1+consts.IDLen for the copy, since it will stop at len(id), but it doesn't really change the functionality to have it vs not so 🤷